src/components/WaitingFor.ts

```
@@ -89,16 +89,19 @@ export const WaitingFor = Vue.component("waiting-for", {
         }
         const input = new PlayerInputFactory().getPlayerInput(createElement, this.players, this.player, this.waitingfor, (out: Array<Array<string>>) => {
             const xhr = new XMLHttpRequest();
+            const root = (this.$root as any);
+            if (root.isServerSideRequestInProgress) return;
```

I would add a `console.warn` here when this request is ignored. If there were ever a bug regarding how `isServerSideRequestInProgress` is being set we would end up failing silently which could be very difficult to track down. With a warning logged we could spot this warning.

---

Makes sense. Not sure I did it correct as I don't see any warning. Please, check 
0107b3d